src/controllers/albumController.js

```diff
@@ -24,6 +55,18 @@ const getSingleAlbum = async (req, res) => {
 };
 
 const updateAlbum = async (req, res) => {
+  /*
+     #swagger.summary = 'Fetch an album  by id and update it'
+     #swagger.parameters['id'] = {
+        description: 'Mongo ObjectID of the album to fetch',
+     }
+     #swagger.responses[200] = {
+				description: 'Album successfully fetched and updated.',
+				schema: { $ref: '#/definitions/Album' }
+		 }
+		 #swagger.responses[404] = { description: 'No album with this id wasfound.' }
+
```

Comment:
For this endpoint, we do `runValidators: true` which means that mongoose will throw a `ValidationError` if the caller tries to pass in any values to update that don't match the schema. Our errorhandler middleware will return a 400 in that case, so we should document that as well

`#swagger.responses[400] = { description: 'validation error' }`